app/views/planning_applications/validation/fee_items/show.html.erb

```diff
@@ -15,6 +15,61 @@
 
     <%= render "fee_item_table" %>
 
+    <% if @planning_application.fee_calculation %>
+      <h3 class="govuk-heading-m">Fee calculation</h3>
+      <table class="govuk-table">
+        <tbody class="govuk-table__body">
+          <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">
+              Application Fee
+            </th>
+            <td class="govuk-table__cell">
+              £<%= @planning_application.fee_calculation.total_fee || 0 %>
+            </td>
+          </tr>
+          <% unless @planning_application.fee_calculation.exemptions.empty? %>
+            <tr class="govuk-table__row">
+              <th scope="row" class="govuk-table__header">
+                Exemptions
+              </th>
+              <td class="govuk-table__cell">
+                <%= @planning_application.fee_calculation.exemptions.map(&:humanize).to_sentence %>
+              </td>
+            </tr>
+          <% end %>
+          <% unless @planning_application.fee_calculation.reductions.empty? %>
+            <tr class="govuk-table__row">
+              <th scope="row" class="govuk-table__header">
+                Reductions
+              </th>
+              <td class="govuk-table__cell">
+                <%= @planning_application.fee_calculation.reductions.map(&:humanize).to_sentence %>
+              </td>
+            </tr>
+          <% end %>
+          <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">
+              Discount
+            </th>
+            <td class="govuk-table__cell">
+              -£<%= @planning_application.fee_calculation.total_fee.to_d - @planning_application.fee_calculation.payable_fee.to_d %>
+            </td>
+          </tr>
+          <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">
+              Total
+            </th>
+            <td class="govuk-table__cell">
+              £<%= @planning_application.fee_calculation.requested_fee || @planning_application.fee_calculation.payable_fee || 0 %>
+            </td>
+            <% if  @planning_application.fee_calculation.requested_fee.present? %>
+              <td>(set by PO)</td>
```

Comment:
I'm not sure if this is strictly correct, as I think not every council has planning officers doing validation? But I don't know what the better generic term would be.

Really this field is just added as a last-minute thing because I think we overlooked the case where an officer has changed the fee and how to display that. Might be worth reviewing it with the designers after the holiday.

Reply:
Rather than set by PO it would probably be better to capture the signed in user? Guess we can do that by either adding a user association or from the relevant audit entry.

Reply:
That's true. I'd thought of the audit but wondered if searching through the audit log might be a mess. I think adding a user field here might make sense, but I wonder if it would be better to wait for design feedback before modifying the database again?